decidim-core/app/commands/decidim/follow_resource.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Decidim
```

Comment:
I like the idea of providing an API to the `Decidim::FollowForm`/`Decidim::CreateFollow` combination, as this is something that's used in other resources (at least 9), but I don't think this would be the best way of handling this case, as I think that for consistency we should have more documentation, specs, inherit from `Rectify::Command`. How about if we just leave it with the old way (explicit FollowForm/CreateFollow) and then if you want and have time, you can create a new PR with these improvements?

Reply:
@andreslucena Can you please check now?

Reply:
Awesome, now it's perfect!!